homeassistant/components/thread/websocket_api.py

```diff
@@ -180,6 +180,9 @@ async def ws_list_datasets(
                 "preferred_border_agent_id": dataset.preferred_border_agent_id,
                 "preferred_extended_address": dataset.preferred_extended_address,
                 "source": dataset.source,
+                "tlv": dataset.tlv,
+                "border_agent_id": "",  # where to get this? (not the preferred border agent ID)
```

Comment:
This is a property of a router. A dataset can't have an extended address as I understand it.
Also, why is this needed, I thought it turns out no one is using it? 🤔